app/js/config.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+const contextPath = window.location.href.split('/')[3];
+const apiBaseUrl = `/${contextPath}/ws/rest/v1`;
+
+export const axiosInstance = axios.create({
+  baseURL: apiBaseUrl,
+  headers: {
+    accept: 'application/json',
```

Comment:
Do we need the last commas?

Reply:
It’s useful for consistency across multiple lines and it's actually added by the linter itself @dkayiwa

Reply:
More explanation can be found [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas)

Reply:
Super cool! Thanks 😊